Bug: heartbeat run CLI event polling fails due to URL encoding (%3F instead of ?)

## Summary

`paperclipai heartbeat run --agent-id <id>` always shows "Internal server error" because the CLI URL-encodes the `?` in the event polling endpoint.

## Actual Behavior

The CLI polls:
```
GET /api/heartbeat-runs/<run-id>/events%3FafterSeq=0&limit=100
```

The `?` is encoded as `%3F`, so the server treats `events%3FafterSeq=0` as a path segment instead of a query parameter. Returns 404/500.

## Expected Behavior

```
GET /api/heartbeat-runs/<run-id>/events?afterSeq=0&limit=100
```

## Impact

The heartbeat run itself IS invoked successfully and the agent executes normally. The bug only affects the CLI's ability to stream events back to the terminal, so users always see "Internal server error" even on successful runs.

This makes `heartbeat run` unusable for debugging or monitoring agent behavior from the CLI.

## Reproduction

```bash
npx paperclipai heartbeat run --agent-id <any-agent-id> --json
# Output: Invoked heartbeat run <id> for agent <name>
#         Internal server error
```

Server log confirms the encoding issue:
```
[ERROR] Unhandled error: GET /api/heartbeat-runs/<id>/events%3FafterSeq=0&limit=100 — Not Found
```

## Environment

- paperclipai CLI via npx
- Node.js v22.22.0
- Linux arm64 (Oracle Cloud)
- Paperclip server: local_trusted mode, embedded Postgres

--- a/cli/src/client/http.ts
+++ b/cli/src/client/http.ts
@@ -104,8 +104,10 @@ export class PaperclipApiClient {
 
 function buildUrl(apiBase: string, path: string): string {
   const normalizedPath = path.startsWith("/") ? path : `/${path}`;
+  const [pathname, query] = normalizedPath.split("?");
   const url = new URL(apiBase);
-  url.pathname = `${url.pathname.replace(/\/+$/, "")}${normalizedPath}`;
+  url.pathname = `${url.pathname.replace(/\/+$/, "")}${pathname}`;
+  if (query) url.search = query;
   return url.toString();
 }
 

